src/safe_apps/models.py

```diff
@@ -10,7 +10,21 @@ def __str__(self) -> str:
         return f"{self.name} | {self.url}"
 
 
+class Client(models.Model):
+    url = models.URLField(
+        unique=True,
+        help_text="The domain URL client is hosted at",
+    )
```

Comment:
Wondering if `models.URLField` is the best field to be used for the constraints that we have on the query side.

would for example: `http://example.com` be different than `https://example.com`.
Or is it relevant that `https://example.com/path/1` is not the same as `https://example.com`

The restrictions set on the access policy seem to be more on the domain level correct? Going with just a pure domain might also be more clear on the query parameter side: `?client_url=http://example.com` vs `?client_url=https://example.com`?

Reply:
This makes sense to me. Is there a pre-defined domain field for django models? Or I should just go with CharField + custom validator?

Reply:
https://code.djangoproject.com/ticket/18120 `wontfix` :/

Reply:
I would say a `CharField` with a custom validator should be enough. If we see that we use the same pattern in more places we can think on creating a custom `Field`.